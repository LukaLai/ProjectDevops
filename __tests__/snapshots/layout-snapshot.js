import { render } from '@testing-library/react'
import Layout from '../../src/app/layout'
import Container from '../../src/app/_components/container';

it('renders layout unchanged', () => {
    // eslint-disable-next-line react/no-children-prop
    const { container } = render(<Layout children={<Container />}/>)
    expect(container).toMatchSnapshot()
})