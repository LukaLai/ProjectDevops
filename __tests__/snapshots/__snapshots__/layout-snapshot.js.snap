// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders layout unchanged 1`] = `
<div>
  <html
    lang="en"
  >
    <head>
      <link
        href="/favicon/apple-touch-icon.png"
        rel="apple-touch-icon"
        sizes="180x180"
      />
      <link
        href="/favicon/favicon-32x32.png"
        rel="icon"
        sizes="32x32"
        type="image/png"
      />
      <link
        href="/favicon/favicon-16x16.png"
        rel="icon"
        sizes="16x16"
        type="image/png"
      />
      <link
        href="/favicon/site.webmanifest"
        rel="manifest"
      />
      <link
        color="#000000"
        href="/favicon/safari-pinned-tab.svg"
        rel="mask-icon"
      />
      <link
        href="/favicon/favicon.ico"
        rel="shortcut icon"
      />
      <meta
        content="#000000"
        name="msapplication-TileColor"
      />
      <meta
        content="/favicon/browserconfig.xml"
        name="msapplication-config"
      />
      <meta
        content="#000"
        name="theme-color"
      />
      <link
        href="/feed.xml"
        rel="alternate"
        type="application/rss+xml"
      />
    </head>
    <body
      class="className"
    >
      <div
        class="min-h-screen"
      >
        <div
          class="container mx-auto px-5"
        />
      </div>
      <footer
        class="bg-neutral-50 border-t border-neutral-200"
      >
        <div
          class="container mx-auto px-5"
        >
          <div
            class="py-28 flex flex-col lg:flex-row items-center"
          >
            <h3
              class="text-4xl lg:text-[2.5rem] font-bold tracking-tighter leading-tight text-center lg:text-left mb-10 lg:mb-0 lg:pr-4 lg:w-1/2"
            >
              Statically Generated with Next.js.
            </h3>
            <div
              class="flex flex-col lg:flex-row justify-center items-center lg:pl-4 lg:w-1/2"
            >
              <a
                class="mx-3 bg-black hover:bg-white hover:text-black border border-black text-white font-bold py-3 px-12 lg:px-8 duration-200 transition-colors mb-6 lg:mb-0"
                href="https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts"
              >
                Read Documentation
              </a>
              <a
                class="mx-3 font-bold hover:underline"
                href="https://github.com/vercel/next.js/tree/canary/examples/blog-starter"
              >
                View on GitHub
              </a>
            </div>
          </div>
        </div>
      </footer>
    </body>
  </html>
</div>
`;
